src/mlpack/core/data/load_arff_impl.hpp

```diff
@@ -219,7 +242,10 @@ void LoadARFF(const std::string& filename,
       {
         std::stringstream error;
         error << "Too many columns in line " << (headerLines + row) << ".";
-        throw std::runtime_error(error.str());
+        if (fatal)
+          Log::Fatal << error.str() << std::endl;
+        else
+          Log::Warn << error.str() << std::endl;
```

Comment:
```suggestion
          Log::Warn << error.str() << std::endl;
          
        return false;
```

I think we need this here to make sure we actually terminate.